src/Compilers/CSharp/Portable/Binder/Semantics/BestTypeInferrer.cs

```diff
@@ -55,7 +55,7 @@ public static NullableFlowState GetNullableState(ArrayBuilder<TypeWithState> typ
             // SPEC:    If no such S exists, the expressions have no best common type.
 
             // All non-null types are candidates for best type inference.
-            IEqualityComparer<TypeSymbol> comparer = conversions.IncludeNullability ? TypeSymbol.EqualsConsiderEverything : TypeSymbol.EqualsIgnoringNullableComparer;
+            IEqualityComparer<TypeSymbol> comparer = conversions.IncludeNullability ? SymbolEqualityComparer.ConsiderEverything : SymbolEqualityComparer.IgnoringNullable;
```

Comment:
>SymbolEqualityComparer.ConsiderEverything : SymbolEqualityComparer.IgnoringNullable [](start = 86, length = 83)

We're comparing `TypeSymbol` through a comparer of `ISymbol` now. Is there a need to change this?